arrange_schedule.py

```diff
@@ -251,11 +247,9 @@ def find_text_acticity(json_obj):
         #reshape deal result
         return_msg["ans_list"] = deal_result
         
-        db.close()  
         return_msg["result"] = "success"
         return return_msg
     except DB_Exception as e:
```

Comment:
We should move `try...except` block into DAO to handle the Exceptions raised as DB execution error in the future.
Now we can just focus on extract SQL, if we do not move the statements out of `try...except` block, or remove the `try...except` block, because the exceptions raised from `mysql` class can still be caught.